packages/admin/blocks-admin/src/blocks/types.tsx

```diff
@@ -152,3 +152,15 @@ export const blockCategoryLabels = {
     [BlockCategory.Form]: <FormattedMessage id="comet.blocks.category.form" defaultMessage="Form" />,
     [BlockCategory.Other]: <FormattedMessage id="comet.blocks.category.other" defaultMessage="Other" />,
 };
+
+export interface LinkBlockInterface<
```

Comment:
Would it be possible to *not* extend the interface and instead implement both? That would make it less a core feature

Reply:
Good idea! Changed it